### DISCUSSION TITLE
엔비디아 드라이버 설치 후  안켜짐

### DISCUSSION THREAD

**Discussion by ShinJinsu** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#117] [2024년도 교육 자료 관련 질문]
엔비디아 드라이버 설치 후 재부팅 시 GNU GRUB 화면에서 *ubuntu 엔터 누르면 좀 진행되다가 -만 깜빡거리면서 리눅스가 켜지지 않습니다.

**Comment by HyeongKeunHong** [comment]
리눅스를 재설치하는게 빠를 것 같습니다.
윈도우로 부팅하신 뒤 하드디스크 파티션 설정창을 여시고, 우분투에 해당하는 볼륨을 삭제하신 뒤, 재설치하시면 됩니다.

재설치 후 영상자료를 따라가실 때에는 엔비디아 드라이버를 설치하는 부분은 생략하고 따라가셔도 무방합니다.
